woocommerce/woocommerce-gateway-stripe
# Ensure notice added with js has same style as notice added with PHP

Fixes #1987

This improves the formatting of the error message shown via js when stripe payment fails. It uses the same classes and wrapper as when the notice is shown from PHP.

Error from js:
![image](https://user-images.githubusercontent.com/1534605/136021671-71916f30-1180-4c7d-ad29-f7f78758cfec.png)
Error from PHP cart notices:
![image](https://user-images.githubusercontent.com/1534605/136021719-b41777f6-6c4e-4e3d-a540-2b4c372d2ac4.png)

Errors from the process card ajax actually will look bad in twenty-twenty-one but this is not related to stripe. 

Probably we should backport this to WCPay too.

# Testing instructions
Checkout with CC # 4000 0027 6000 3184 and click fail authorization. Check that the error message appears at the top of the page and looks good in several themes. Style should be the same as checkout with a redirect payment method like bancontact and clicking fail authorization on the stripe page.

-------------------
